docker-startup.sh

```diff
@@ -2,11 +2,18 @@
 set -euo pipefail
 [[ "${DEBUG:-false}" == "true" ]] && set -x
 
-if [[ ! -d "node_modules" ]]; then
+if [[ ! -d "./node_modules" ]]; then
   npm install
 fi
 
+apt update -y
+apt --fix-broken install -y
+apt install -y ca-certificates fonts-liberation libasound2 libatk-bridge2.0-0 libatk1.0-0 libc6 libcairo2 libcups2  libdbus-1-3 libexpat1 libfontconfig1 libgbm1 libgcc-s1 libglib2.0-0 libgtk-3-0 libnspr4 libnss3 libpango-1.0-0 libpangocairo-1.0-0 libstdc++6 libx11-6 libx11-xcb1 libxcb1 libxcomposite1 libxcursor1 libxdamage1 libxext6 libxfixes3 libxi6 libxrandr2 libxrender1 libxss1 libxtst6 lsb-release wget xdg-utils
+dpkg --configure -a
+
```

Comment:
Are these actually required. These are packages to have access to a X11 / Graphics environment. Guess that are required for puppeteer, right ? If so, we should address ASAP to just open a simple page with a print CSS  to facilitate the PDF / printing generation of user tokens

Reply:
removed as not needed anymore